mob.go

```diff
@@ -1009,12 +1015,11 @@ func done(configuration Configuration) {
 		if err != nil {
 			sayError(err.Error())
 		}
-		if configuration.MobDoneSquash {
-			if isNothingToCommit() {
-				sayInfo("nothing was done, so nothing to commit")
-			} else {
-				sayTodo("To finish, use", "git commit")
-			}
+
+		if isNothingToCommit() {
+			sayInfo("nothing was done, so nothing to commit")
```

Comment:
When doing a merge, this message is wrong. We need two different messages, one for each option.

Reply:
In case of `--no-squash` we could basically omit it. That would be fine as well.